locales/en/messages.json

```diff
@@ -1512,10 +1512,7 @@
         "message": "Dynamic Notch values change"
     },
     "dialogDynFiltersChangeNote": {
-        "message": "<span class=\"message-negative\"><b>WARNING: Some dynamic notch values have been changed to default values</b></span> because the RPM filtering has been activated/deactivated.<br> Please, check before flying."
-    },
-    "dialogDynFiltersConfirm": {
-        "message": "OK"
+        "message": "<span class=\"message-negative\"><b>WARNING: This change will enable/disable RPM filtering, increasing/decreasing filter delay.</b></span><br>Reset the dynamic notch filters to recommended values?"
```

Comment:
One single message is fine.
Maybe add the word `/effectiveness`, so it reads: `increasing/decreasing filter delay/effectiveness.`

Reply:
I will change it :)